src/store/Store.js

```diff
@@ -69,6 +79,22 @@ export default class Store {
     this._dispatchToken = this._dispatcher.register(
       this._handleDispatch.bind(this)
     );
+
+    if (hasReduxDevTools) {
+      this._devToolsExtension = window.__REDUX_DEVTOOLS_EXTENSION__.connect({
+        name: `${this._name}_${this._uid}`,
```

Comment:
Do you think it'd be safe to only include the _uid if there's no name?

Reply:
Name's can be repeated safely so we can definitely omit it if the name `!== 'Store'`